tests/tracing/test_trace_all.py

```diff
@@ -18,8 +18,8 @@ def main():
 
 
 def test_everything_is_traced():
-    called = decorators.entrypoint(None)(main)
-    assert called is not None
+    trace_data, _ = decorators.entrypoint(None)(main)
```

Comment:
What does this do? the `entrypoint` still returns only a `DataFrame`, right?

Reply:
Additionally, we also return the performance data. This is required for test_evaluation_decorators.py.